lib/image_optim/worker/jhead.rb

```diff
@@ -7,6 +7,8 @@ class Worker
     #
     # Jhead internally uses jpegtran which should be on path
     class Jhead < Worker
+      TIMEOUT_OPTION = timeout_option
```

Comment:
From what I understand, none of the image optimizing is done in place right? Even for `optimize_image!`, it does not replace the original image if optimization fails.